src/components/date.js

```diff
@@ -32,7 +32,7 @@ export default class FormattedDate extends Component {
     }
 
     render() {
-        const {formatDate}      = this.context.intl;
+        const {formatDate, textComponent} = this.context.intl;
```

Comment:
I guess I don't understand this. if the textComponent is of type Text, it would become RN specific. Did I misunderstand something? Shouldn't it be of type `node`?